physlr/physlr.py

```diff
@@ -1468,7 +1468,13 @@ def detect_communities_cosine_of_squared(g, node_set, squaring=True, threshold=0
         import numpy as np
         from sklearn.metrics.pairwise import cosine_similarity
 
+        if len(node_set) < Physlr.args.skip_small:
```

Comment:
This is the skip small components to avoid oversplitting that you were talking about, right?

Reply:
yep. when the subgraph is too small, the cosine similarity metric by nature does not work well. The default value is now set to 10 which worked well (in terms of not doing oversplitting) in December when I was testing.

Reply:
Why? the function there is file indication a subset of barcodes (vertices), this function reads in those vertices and considers only those vertices fo stat retrieval. but if there's no input file for that, it works on the whole graph..